manila/share/manager.py

```diff
@@ -118,6 +118,10 @@
                     'the share manager will poll the driver to perform the '
                     'next step of migration in the storage backend, for a '
                     'migrating share server.'),
+    cfg.BoolOpt('server_migration_extend_neutron_network',
```

Comment:
That's good, when we try to bring this upstream - it should be explained that this can only be enabled if neutron supports the respective `binding-extended` plugin (and our change for manila ports has to be brought upstream, too, beforehand then)

Maybe add a comment about that?

Reply:
help updated, adding that the Neutron should support of multiple port bindings